great_expectations/render/renderer/content_block/bullet_list_content_block.py

```diff
@@ -107,67 +150,243 @@ def expect_column_unique_value_count_to_be_between(cls, expectation, column_name
                 }
             }]
 
+    # NOTE: This method is a pretty good example of good usage of `params`.
+    # NOTE: I think `column_name` should be deprecated.
     @classmethod
     def expect_column_values_to_be_between(cls, expectation, column_name=""):
-        if (expectation["kwargs"]["min_value"] is None) and (expectation["kwargs"]["max_value"] is None):
+        params = substitute_none_for_missing(
+            expectation["kwargs"],
+            ["column", "min_value", "max_value", "mostly"]
+        )
+
+        if (params["min_value"] is None) and (params["max_value"] is None):
+            # Note: I'm not sure why we're using a different pattern for templating column names...
+            # This would probably be better as `"$column has a bogus $expectation_name expectation.",`
+
+            # Related issue: are we going to want to allow for different styles of parameters?
+            # For example, I could imagine wanting one color of syntax highlighting for variable names,
+            # and a different color for expectations.
+            # In that case, perhaps returning something like this would work?
+            # {
+            #     "template": "$column has a bogus $expectation_name expectation.",
+            #     "params": {
+            #         "column": "my_column",
+            #         "expectation_name": "expect_column_values_to_be_between",
+            #     },
+            #     "param_types": {
```

Comment:
A new type system!!! We should see how many different type systems we can collect inside GE. :)

Reply:
Seriously though, without a corresponding understanding of what would happen in the view it's hard to track exactly what you're thinking here. I like the idea of the power of differentiating in a view, but also think bottom line is that the template-in-template design we have in the view now is a stress (== illuminating and useful) for design of views in terms of content-block vs renderer distinction.

Reply:
Yeah, a small prototype/demo will be helpful here. I'll knock that out.